pkg/cmd/server/server.go

```diff
@@ -108,7 +108,11 @@ func main() {
 				srv := jrpc2.NewServer(assigner, servOptions)
 				handler.SetConnection(srv)
 				srv.Start(ch)
-				go func() { defer cancel(); srv.Wait() }()
+				go func() {
```

Comment:
the changes are not related to the libovsdb

Reply:
yup, rebased (and removed this)